assign_rights/tests.py

```diff
@@ -50,6 +52,31 @@ def test_grouping_form(self):
                 form.errors[field], False,
                 "Field-specific error message not raised for {}".format(field))
 
+    def test_rightshell_views(self):
```

Comment:
We can leave this for now,  but I think eventually we'll want to refactor these tests because they're basically the same for RightsShells and Groupings.

Reply:
Cool--I also want to address rights granted or restricted in these tests in future iterations.